ios/Mattermost.xcodeproj/project.pbxproj

```diff
@@ -828,8 +828,9 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = Mattermost/Mattermost.entitlements;
-				CODE_SIGN_IDENTITY = "iPhone Developer";
+				CODE_SIGN_IDENTITY = "Apple Development";
```

Comment:
Avinash, revert changes to this file please

Reply:
I'm not sure how to do it. Can you please guide me ?

Reply:
sure just do `git checkout gekidou -- ios/Mattermost.xcodeproj/project.pbxproj` on this branch and that should revert the change, then you can push the commit

Reply:
done